roles/tas_single_node/vars/main.yml

```diff
@@ -90,3 +90,48 @@ tas_single_node_remote_tsa_signer_private_key: "{{ tas_single_node_certs_dir }}/
 tas_single_node_remote_tsa_leaf_certificate: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_tsa_leaf_certificate_filename }}"
 tas_single_node_remote_tsa_certificate_chain: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_tsa_certificate_chain_filename }}"
 tas_single_node_remote_tsa_private_key: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_tsa_private_key_filename }}"
+
+# Pod/Service names + Ports
+tas_single_node_cli_server_pod: cli-server
+tas_single_node_cli_server_port: 8080
+
+tas_single_node_ctlog_pod: ctlog
+tas_single_node_ctlog_port1: 6962
```

Comment:
I think instead using "port1", "port2" etc, we should name these ports - e.g. is this a gRPC port, http port, something else? I assume they all have separate different usage and naming them based on that is more useful.